g11n-ws/modules/md-service-i18n-l3/src/main/java/com/vmware/vip/core/messages/service/product/ProductService.java

```diff
@@ -166,8 +167,31 @@ private DropVersionDTO getDropVersion(DropVersionDTO versioninfo,JSONObject jo,
 						+ componentMessagesDTO.getVersion(), e);
 			}
 		}
+		updateMultComponentCache(componentMessagesDTOList.get(0).getProductName(), componentMessagesDTOList.get(0).getVersion());
```

Comment:
Is it necessary to judge length of componentMessagesDTOList greater than 0?

Reply:
It is not necessary. because have not null validation